songs/api/SongPermission.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import permissions
+
+
+class SongCreatePermission(permissions.BasePermission):
+    """
+    Permission to check if the user is admin or not
+    """
+
+    def has_permission(self, request, view):
```

Comment:
what do we return here if user is not admin?

Reply:
I added this check for anonymous user, I have modified it. Now it will return False if user is not admin